web-ui/src/main/resources/catalog/components/viewer/partials/mainviewer.html

```diff
@@ -176,6 +182,12 @@ <h3>{{'ManageLayers' | translate}}
     </div>
   </div>
 
+  <div class="panel panel-default panel-tools"
+       ng-show="activeTools.legend">
+    <div class="panel-carousel" gn-legend-panel="map"></div>
```

Comment:
I would add a title and close button to this panel (similar to the other ones), as well as a text when no layer is added to the map, if possible.

Reply:
Done by Michel in https://github.com/geonetwork/core-geonetwork/pull/2997/files#diff-d2d5d727728ce4190d2c11d51973cb7eR187 and
https://github.com/geonetwork/core-geonetwork/pull/2997/files#diff-dacf5664739f9ad2672a5f03ba06c19aR14